Helper/SupportFormHelper.php

```diff
@@ -0,0 +1,131 @@
+<?php
+
+namespace Adyen\Payment\Helper;
+
+use Magento\Framework\App\ProductMetadataInterface;
+use Magento\Framework\Exception\NoSuchEntityException;
+use Magento\Framework\Mail\Template\TransportBuilder;
+use Magento\Framework\Message\ManagerInterface as MessageManagerInterface;
+use Magento\Store\Model\StoreManagerInterface;
+
+class SupportFormHelper
+{
+    /**
+     * @var Config
+     */
+    protected Config $config;
```

Comment:
```suggestion
    protected $config;
```

Type declaration in this file should be removed if we are to offer this functionality on v8, since we are still supporting php 7.3 on it

Reply:
So this should be done for all of the types in this file  and the others too, right ? @Morerice